rootfs/qubes.SetAudioVM

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -e
+
+# Default to dom0 if called with empty arg
+domid="${1:-0}"
+
+export PULSE_SERVER=unix:/tmp/pa.sock
+
+pactl unload-module module-vchan-sink
```

Comment:
wouldn't this fail if changing from none to something? the module is not loaded at that time...

Reply:
It looks like unloading module that is not loaded already still work and exit with 0.

Reply:
Looks like pactl exits with 0 if module is already unloaded...

Reply:
For example, in the stubdom shell:
```
pactl unload-module module-vchan-sink
Failed to load cookie file from cookie: No such file or directory
Failed to unload module: Module module-vchan-sink not loaded
echo $?
0
pactl unload-module module-vchan-sink
Failed to load cookie file from cookie: No such file or directory
Failed to unload module: Module module-vchan-sink not loaded
echo $?
0
```